dashboard/fixtures/certificates.json

```diff
@@ -5,14 +5,14 @@
 	    "certificate_type": "verified",
 	    "status": "downloadable",
 	    "download_url": "http://www.example.com/demo.pdf",
-	    "grade": "0.98"
+	    "grade": 98
```

Comment:
I think it's fine to have our Elasticsearch data be different than what we store in the JSONField in the model. For example we may want to create an extra field for customized sorting in Elasticsearch. Since this JSON is supposed to represent data coming from edX we shouldn't make any change here. I'm fine with changing `grade` to an int in the serializer function, I think it's less effort than upgrading Elasticsearch, and I'm not sure it's wise to point to an alpha version. Alternatively we can also leave the field alone and create some other field like `grade_histogram` which represents the grade as an int.